[All versions] Changed constructor section

pkosko opened the pull request:
* improved the way how the constructors are described, it shows constructors in
a separated section and allow adding brief for them (especially useful for multiple
constructors). This change does only affect visual of 'Constructors' section, no other changes are done.

mijin85cho commented:
Just a question. It looks like the existing Constructor descriptions are improved.

As the new h4 titiles also are added to existing h4 titles, now for the Constructor section we have 2 titles.

For example on 5.5/device_api/wearable/tizen/tizen.html
![image](https://user-images.githubusercontent.com/9958374/69107842-115f3800-0ab6-11ea-87e6-ef830abc9652.png)


![image](https://user-images.githubusercontent.com/9958374/69107928-6307c280-0ab6-11ea-8014-baa5acdda2e4.png)

Is it ok to have doubled titles?

pkosko commented:
@mijin85cho thank you for your comment, we need to check this about duplication of h4 headers, so I extracted application-remark-related change to different PR #947 if you can, please accept it.

About constructor additional header we added, it was desired to have 'Constructors' header as the begin of the constructors list, which is better visible here:
![image](https://user-images.githubusercontent.com/38070006/69123077-ba9b3380-0aa0-11ea-9135-7caa0f0f45d3.png)

I only wonder if using h4 is the best as you commented, we need to consider changing header types in this change for consistency.
For now, please ignore this PR, I will let you know about progress.

pkosko commented:
@mijin85cho I updated this commit regarding to your comment. Now, the h4 header for each constructor was replaced with the same style as for every function in Web reference, for consistency purposes. The result is as below:
![image](https://user-images.githubusercontent.com/38070006/69140044-01e5ec00-0ac2-11ea-8fbf-529d4ef0f44a.png)

pkosko commented on docs/application/web/api/5.5/device_api/wearable/tizen/cordova/media.html:
```diff
@@ -146,11 +146,17 @@ <h2>1. Interfaces</h2>
               </div>
 <pre class="webidl prettyprint">Media(DOMString src, optional <a href="cordova.html#SuccessCallback">SuccessCallback</a>? successCallback, optional <a href="#MediaErrorCallback">MediaErrorCallback</a>? errorCallback,
       optional <a href="#StatusChangeCallback">StatusChangeCallback</a>? mediaStatus);</pre>
+<p><span class="version">Since: </span>
```
Each API element can be introduced since some version. Line 157 is related to Constructor. Line 193 is 'since' related to Constants section. In this conditions, the behaviour is correct.

pkosko commented on docs/application/web/api/5.5/device_api/wearable/tizen/notification.html:
```diff
@@ -1616,7 +1616,12 @@ <h4>Dictionary members</h4>
 <h4 id="StatusNotification::constructor">Constructors</h4>
-<dl><pre class="webidl prettyprint"><span class="nocode deprecated">StatusNotification(<a href="#StatusNotificationType">StatusNotificationType</a> statusType, DOMString title, optional <a href="#StatusNotificationInit">StatusNotificationInit</a>? notificationInitDict);</span></pre></dl>
+<dl>
+<dt class="method" id="">
```
Constructor, the same as each member of StatusNotification interface was deprecated 'by parent', not directly, so the style is not 'deprecated'. This is consistent with not using 'deprecated' style for members of the StatusNotification interface, which are not 'grayed' also.
